### PR TITLE
update example/walkthrough to v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
I have verified that all the v1beta3 yaml files are working properly @nikhiljindal

### CODE REVIEW

File: examples/walkthrough/README.md [hunk 1]
```diff
@@ -29,49 +27,39 @@ See the [design document](../../DESIGN.md) for more details.
...
+          # mount path within the container
+          mountPath: /data/redis
+  volumes:
+    - name: redis-persistent-storage
+      emptyDir: {}
 ```
 
-Ok, so what did we do?  We added a volume to our pod:
-
+Ok, so what did we do? We added a volume of our pod:
```
Comment (nikhiljindal) [inline examples/walkthrough/README.md]: Adding a volume "to" pod seems better than volume "of" pod

File: examples/walkthrough/k8s201.md [hunk 2]
```diff
@@ -21,55 +21,54 @@ OK, now you have an awesome, multi-container, labelled pod and you want to use i
...
+apiVersion: v1beta3
 kind: ReplicationController
-desiredState:
+metadata:
+  name: nginx-controller
+spec:
   replicas: 2
-  # replicaSelector identifies the set of Pods that this
+  # selector identifies the set of Pods that this
   # replicaController is responsible for managing
```
Comment (nikhiljindal) [inline examples/walkthrough/k8s201.md]: replicaController -> replication controller

File: examples/walkthrough/k8s201.md [hunk 3]
```diff
@@ -21,55 +21,54 @@ OK, now you have an awesome, multi-container, labelled pod and you want to use i
 Replication controllers are the objects to answer these questions.  A replication controller combines a template for pod creation (a "cookie-cutter" if you will) and a number of desired replicas, into a single API object.  The replica controller also contains a label selector that identifies the set of objects managed by the replica controller.  The replica controller constantly measures the size of this set relative to the desired size, and takes action by creating or deleting pods.  The design of replica controllers is discussed in detail [elsewhere](http://docs.k8s.io/replication-controller.md).
 
 An example replica controller that instantiates two pods running nginx looks like:
```
Comment (nikhiljindal) [inline examples/walkthrough/k8s201.md]: replicaController -> replication controller

**Comment by nikhiljindal** [comment]
LGTM, modulo a few nits.
You need to fix tests (Shippable is red)

You should also update the PR title to something more appropriate like "Updating examples/walkthrough to use v1beta3 configs"

**Comment by caesarxuchao** [comment]
@nikhiljindal 
updated, wait for shippable to be green

**Comment by nikhiljindal** [comment]
Shippable is still red.

**Comment by caesarxuchao** [comment]
@nikhiljindal  Do you know why the first build failed but the second on was successful?

**Comment by caesarxuchao** [comment]
I didn't change anything but shippable is green now

**Comment by nikhiljindal** [comment]
It had timedout last time. I restarted it and it is green now.

File: examples/walkthrough/k8s201.md [hunk 4]
```diff
@@ -125,31 +124,27 @@ In all cases, if the Kubelet discovers a failure, the container is restarted.
...
+spec:
+  containers:
+    - name: nginx
+      image: nginx
+      # defines the health checking
+      livenessProbe:
+        # an http probe
+        httpGet:
+          path: /_status/healthz
+          port: 8080
```
Comment (bgrant0607) [inline examples/walkthrough/k8s201.md]: Doesn't match the exposed containerPort (80)

File: examples/walkthrough/pod-with-http-healthcheck.yaml [hunk 5]
```diff
@@ -1,24 +1,20 @@
...
+spec:
+  containers:
+    - name: nginx
+      image: nginx
+      # defines the health checking
+      livenessProbe:
+        # an http probe
+        httpGet:
+          path: /_status/healthz
+          port: 8080
```
Comment (bgrant0607) [inline examples/walkthrough/pod-with-http-healthcheck.yaml]: Same problem

Comment (caesarxuchao) [inline examples/walkthrough/pod-with-http-healthcheck.yaml]: Thanks for the pick.

**Pull request merged**